test/index.spec.ts

```diff
@@ -0,0 +1,128 @@
+import nock from 'nock';
```

Comment:
As default unit tests shouldn't give a fuc* for api response. Our responsibility it's guarantee that our code supply necessity. I think less is more in this case! At the place of create REAL PERSISTS, use jest to mock response and work with it!

Reply:
Nock in my perspective it's for integration tests 🤔

Reply:
As a matter of fact, we don't care about the response. The idea of these tests is to guarantee that we're calling the correct endpoint with the correct params and nothing else.

Reply:
If you work with a function to manipulate data, a function to build a client and a function to call endpoints. You could mock deeps and test all of this stuff without need save nock files and make test hit API once to lead with the return.